--- rlpr-2.06/configure.in.orig	2005-06-06 12:08:38.000000000 +0200
+++ rlpr-2.06/configure.in	2005-06-06 12:10:01.000000000 +0200
@@ -103,7 +103,7 @@
 AC_CHECK_FUNCS(strchr getopt_long strcasecmp strtoul strtol vsyslog)
 
 AC_REPLACE_FUNCS(strdup strstr strcasecmp)
-AC_CHECK_FUNC(getopt_long,, AC_LIBOBJ([getopt1]) AC_LIBOBJ([getopt]))
+AC_CHECK_FUNC(getopt_long,, [AC_LIBOBJ([getopt1]) AC_LIBOBJ([getopt])])
 
 AC_CONFIG_FILES([Makefile lib/Makefile src/Makefile intl/Makefile man/Makefile po/Makefile.in])
 AC_OUTPUT
