--- rlpr-2.04/configure.in~	Thu Jul 25 06:34:18 2002
+++ rlpr-2.04/configure.in	Thu Jul 25 06:34:18 2002
@@ -23,9 +23,6 @@
 dnl Set of available languages
 ALL_LINGUAS=
 
-dnl LIBOBJS builds a list of all the helper source files we need
-LIBOBJS=
-AC_SUBST(LIBOBJS)
 
 dnl Checks for programs.
 AC_PROG_INSTALL
@@ -110,6 +110,6 @@
 AC_CHECK_FUNCS(strchr getopt_long strcasecmp strtoul strtol)
 
 AC_REPLACE_FUNCS(strdup strstr strcasecmp)
-AC_CHECK_FUNC(getopt_long,, [LIBOBJS="$LIBOBJS getopt1.o getopt.o"])
+AC_CHECK_FUNC(getopt_long, [], [AC_LIBOBJ([getopt1]) AC_LIBOBJ([getopt])])
 
 AC_OUTPUT([Makefile lib/Makefile src/Makefile intl/Makefile man/Makefile po/Makefile.in], [sed -e "/POTFILES =/r po/POTFILES" po/Makefile.in > po/Makefile])
